libs/codegen/src/future/index.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * `@cugetreg/codegen/future` are types that we use
+ * but does not available yet in backend's GraphQL.
+ *
+ * TODO: Wait for backend to implement these types into GraphQL
+ */
+
+export enum Semester {
```

Comment:
? There is no need to add future/index.ts because, in the end, we will probably use it anyway.

Reply:
temporary

Reply:
u could just add Semester enum in graphql right now btw in course.enum.graphql in cugetreg-api